src/modules/users/Config.cpp

```diff
@@ -656,6 +657,59 @@ Config::setRootPasswordSecondary( const QString& s )
     }
 }
 
+void
+Config::setActiveDirectoryUsed( bool used )
+{
+    m_activeDirectoryUsed = used;
+}
+
+bool
+Config::getActiveDirectoryEnabled() const
+{
+    return m_activeDirectory;
+}
+
+bool
+Config::getActiveDirectoryUsed() const
+{
+    return m_activeDirectoryUsed && m_activeDirectory;
+}
+
+void
+Config::setActiveDirectoryAdminUsername( const QString & s )
+{
+    m_activeDirectoryUsername = s;
+}
+
+void
+Config::setActiveDirectoryAdminPassword( const QString & s )
```

Comment:
This API (the others, too) seems a bit peculiar to me because the name of the method says "Admin" but the data member does not. Whose password **is** it then?